students/joel.json

```diff
@@ -0,0 +1,3 @@
+{ "name": "Jo-el",
+  "last meal": "DUMPS!!"
+}
```

Comment:
My bad. I had already merged into my local master. So this branch got both commits.